apps/angular/simple-animations/src/app/app.config.ts

```diff
@@ -1,5 +1,8 @@
 import { ApplicationConfig } from '@angular/core';
+import { provideAnimations } from '@angular/platform-browser/animations';
 
 export const appConfig: ApplicationConfig = {
-  providers: [],
+  providers: [provideAnimations()],
 };
+
+// provideAnimationsAsync won't enable animations on load
```

Comment:
That's an interesting information. But indeed, we should use async only to delay the load of the package. If we know that we need the animation right away, that's normal to load the package immedialty